app/pages/lab/workflows-table.jsx

```diff
@@ -57,7 +57,7 @@ const WorkflowsTable = ({
                   Active
                 </label>
               </td>
-              <td>
+              <td class="stats_completeness_type">
```

Comment:
Non-blocking, but we were using BEM convention last on PFE, so that'd still be useful I think.

Reply:
I took a guess at how PFE is using BEM and went for `standard-table__label__completeness` as the class name. Unfortunately, I don't think we documented how we were using BEM.

Reply:
I'm not sure what would help regarding documentation, however, using the BEM standard, the block would be `standard-table`, the element is `label`, and `completeness` would be the modifier, so `standard-table__label--completeness` is how I'd do it.